src/main/java/org/zalando/nakadi/view/TimelineView.java

```diff
@@ -0,0 +1,67 @@
+package org.zalando.nakadi.view;
+
+import org.zalando.nakadi.domain.Timeline;
+
+import java.util.Date;
+import java.util.UUID;
+
+public class TimelineView {
+
+    private final UUID id;
+    private final String eventType;
+    private final int order;
+    private final String storageId;
+    private final String topic;
+    private final Date createdAt;
+    private final Date switchedAt;
+    private final Date cleanupAt;
```

Comment:
👍 . There are a few occurrences of `cleanupAt`, but they're only in tests, so I think this is fine.